python/setup.py

```diff
@@ -499,11 +499,6 @@ def _move_shared_libs_unix(build_prefix, build_lib, lib_name):
     lib_filename = os.path.basename(libs[0])
     shutil.move(pjoin(build_prefix, lib_filename),
                 pjoin(build_lib, 'pyarrow', lib_filename))
-    for lib in libs[1:]:
-        filename = os.path.basename(lib)
-        link_name = pjoin(build_lib, 'pyarrow', filename)
-        if not os.path.exists(link_name):
-            os.symlink(lib_filename, link_name)
```

Comment:
If the user wants to link a C++ library to the libarrow that is bundled with wheels, will it still work if there is only `libarrow.so.14` there?

Reply:
Tensorflow is doing the same. Downstream users might need to add something to their linker commands though.

Reply:
Side note, when I extracted the tensorflow packages, I noted they used the versionless path.